app-misc/keyd/files/keyd-2.5.0-makefile.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/Makefile b/Makefile
+index b4fc2bc..828b649 100644
+--- a/Makefile
++++ b/Makefile
+@@ -2,7 +2,7 @@
+ VERSION=2.5.0
+ COMMIT=$(shell git describe --no-match --always --abbrev=7 --dirty)
+ VKBD=uinput
+-PREFIX?=/usr/local
++PREFIX=$(EPREFIX)/usr
```

Comment:
Typically you'd pass `PREFIX="${EPREFIX}/usr"` as an argument to `emake install` instead of patching the Makefile. Also consider pleading with upstream to use a real build system, such as Meson or autoconf.